apps/src/code-studio/components/progress/ScriptOverviewHeader.jsx

```diff
@@ -68,6 +70,15 @@ class ScriptOverviewHeader extends Component {
             announcements={verifiedResourcesAnnounce.concat(announcements)}
           />
         }
+        {showVersionWarning &&
+          <Notification
+            type={NotificationType.warning}
+            notice={i18n.wrongCourseVersionWarningNotice()}
+            details={i18n.wrongUnitVersionWarningDetails()}
+            dismissible={true}
+            width={1100}
```

Comment:
I've extracted a constant for this. I'm ignoring where it's used in TeacherNotification.jsx though, because it looks like that whole file can be removed (in a separate PR).

the current width behavior could be improved by using max-width instead of width, since that will allow the announcement to display narrower on smaller screens along with the rest of the content. however I don't want to change too many things before this week's deadline arrives, so that's also going onto my list of follow-up tasks for after June 4.